sormas-rest/src/main/java/de/symeda/sormas/rest/CaseResource.java

```diff
@@ -129,4 +130,10 @@ public CaseDataDto getByUuid(@PathParam("uuid") String uuid) {
 		return FacadeProvider.getCaseFacade().getByUuid(uuid);
 	}
 
+	@POST
+	@Path("/patchJson/{uuid}")
+	public CaseDataDto patchJson(@PathParam("uuid") String uuid, JSONObject jsonObject) throws Exception {
```

Comment:
Suggestion: postUpdate
Also add a comment that explains how/when to use this and what the JSON object should look like